src/ComplaintDetail.jsx

```diff
@@ -86,46 +87,62 @@ export class ComplaintDetail extends React.Component {
     return (
       <button className="a-btn a-btn__link"
               onClick={() => { window.location = root }}>
-          <span className="cf-icon cf-icon-left"></span>
+          {/* Includes custom cf-icon-left class to set color. */}
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 559.6 1200" className="cf-icon-svg cf-icon-left"><path d="M494.5 1090.7c-17.3 0-33.8-6.8-46-19L19 642.1c-25.4-25.4-25.4-66.5 0-91.9l429.5-429.5c25.6-25.1 66.8-24.8 91.9.8 24.8 25.3 24.8 65.8 0 91.1L156.9 596.2l383.6 383.6c25.4 25.4 25.4 66.5.1 91.9-12.3 12.2-28.8 19-46.1 19z"></path></svg>
           Go to search home page
       </button>
     )
   }
 
   _renderCompanyTimely( value ) {
-    const styles = [ 'cf-icon', 'cf-icon__before', 'cf-icon-clock-round' ]
+    const styles = [ 'cf-icon__before' ]
     if ( value.toLowerCase() === 'no' ) {
       styles.push( 'not-timely' )
     }
 
     return (
       <div>
-        <span className={styles.join( ' ' )}></span>
+        <span className="cf-icon__before">
+          {/*
+            Includes custom cf-icon-clock-round
+            and not-timely (set via condition) class to set color.
+          */}
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1200" class="{ 'cf-icon-svg cf-icon-clock-round ' + ( value.toLowerCase() === 'no' ? 'not-timely' : '' ) }"><path d="M606.1 354.2c-139.1-59-299.8 5.9-358.8 145.1s5.9 299.8 145.1 358.8c139.1 59 299.8-5.9 358.8-145.1 43.6-102.7 20.5-221.5-58.4-300.4-24.9-24.9-54.3-44.7-86.7-58.4zM636 542.6L517.9 660.7c-.3.3-.6.5-.8.8l-.5.4-.5.4-.6.5-.4.3-.7.5-.3.2-.7.4-.3.2-.7.4-.4.2-.6.3-.5.2-.6.3-.5.2-.5.2-.6.2-.5.2-.7.2-.5.2-.8.2-.4.1-.8.2-.4.1-.8.1-.5.1-.7.1H497.8l-.7-.1-.5-.1-.8-.1-.4-.1-.8-.2-.4-.1-.8-.2-.4-.1-.7-.2-.5-.2-.6-.2-.5-.2-.5-.2-.6-.3-.5-.2-.6-.3-.4-.2-.7-.4-.4-.2-.7-.4-.4-.2-.6-.5-.4-.3-.6-.5-.5-.4-.5-.4c-.3-.3-.6-.5-.9-.8-.3-.3-.5-.6-.8-.8l-.4-.5-.4-.5-.5-.6-.3-.4c-.2-.2-.3-.4-.5-.7l-.2-.3c-.1-.2-.3-.5-.4-.7l-.2-.3c-.1-.2-.3-.5-.4-.7l-.2-.4c-.1-.2-.2-.4-.3-.7l-.2-.4-.3-.6-.2-.5-.2-.5c-.1-.2-.2-.4-.2-.6l-.2-.5c-.1-.2-.1-.5-.2-.7l-.1-.4c-.1-.2-.1-.5-.2-.8s-.1-.3-.1-.4-.1-.5-.2-.8 0-.3-.1-.4-.1-.5-.1-.8 0-.3-.1-.5-.1-.5-.1-.7v-.7-.6V414c0-13.8 11.2-25 25-25s25 11.2 25 25V582.8l75.4-75.4c9.8-9.8 25.6-9.8 35.4 0s9.8 25.6 0 35.4l-.1-.2z"></path><path d="M500 105.2c-276.1 0-500 223.9-500 500s223.9 500 500 500 500-223.9 500-500-223.9-500-500-500zm0 822.7c-178.1 0-322.5-144.4-322.5-322.5S321.9 282.8 500 282.8s322.5 144.4 322.5 322.5S678.1 927.9 500 927.9z"></path></svg>
+        </span>
         <span className="body-copy">{ value }</span>
       </div>
     )
   }
 
   _renderConsumerConsent( value ) {
+    // Each SVG includes custom classes for setting color.
```

Comment:
The iconMap should probably be generated in the constructor (like the renderMap) so the svg elements are only created once and reused on further renders.